### PR TITLE
chore: enable prettier for JavaScript sources

### PR DESCRIPTION
**Pull request by bajtos**
## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "lint:fix": "npm run prettier:fix && npm run tslint:fix",
     "tslint": "node packages/build/bin/run-tslint --project tsconfig.json",
     "tslint:fix": "npm run tslint -- --fix",
-    "prettier:cli": "node packages/build/bin/run-prettier \"**/*.ts\"",
+    "prettier:cli": "node packages/build/bin/run-prettier \"**/*.ts\" \"**/*.js\"",
```
Comment (bajtos) [inline package.json]: This is the actual change in our build scripts.

File: packages/cli/generators/example/downloader.js [hunk 2]
```diff
@@ -8,7 +8,7 @@
 const path = require('path');
 const pacote = require('pacote');
 
-module.exports = function downloadAndExtractExample(exampleName, cwd) {
+module.exports = async function downloadAndExtractExample(exampleName, cwd) {
```
Comment (bajtos) [inline packages/cli/generators/example/downloader.js]: Ouch! I was incorrectly using `await` in a function that was not marked as `async` before. This is not valid a JavaScript, although it really makes me wonder how is it possible that are tests are still passing.

File: packages/cli/generators/example/index.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ module.exports = class extends BaseGenerator {
     );
   }
 
-  downloadAndExtract() {
+  async downloadAndExtract() {
```
Comment (bajtos) [inline packages/cli/generators/example/index.js]: Ditto - `await` is used in a function that was not `async`.

File: packages/cli/generators/example/downloader.js (hunk 2 above)
Comment (virkt25) [reply packages/cli/generators/example/downloader.js]: Interesting ... does it mean the test is possibly incorrect?

Comment (bajtos) [reply packages/cli/generators/example/downloader.js]: Technically, this source code is an invalid JavaScript that should have produced parsing error when loaded by V8. My guess is that V8 is perhaps clever enough to detect presence of `await` keyword and mark the function as `async` automatically? IDK.

**Pull request merged**